regulations/static/regulations/js/source/views/main/child-view.js

```diff
@@ -77,8 +77,9 @@ var ChildView = Backbone.View.extend({
     render: function() {
         this.updateWayfinding();
         this.loadImages();
-        HeaderEvents.trigger('section:open', this.id);
-        DrawerEvents.trigger('section:open', this.id);
+        // TODO: What is this for?
+        // HeaderEvents.trigger('section:open', this.id);
+        // DrawerEvents.trigger('section:open', this.id);
```

Comment:
`HeaderEvents` is needed for non-reg-view views. The `RegView` [calls](https://github.com/eregs/regulations-site/blob/master/regulations/static/regulations/js/source/views/main/reg-view.js#L50) the same event, but the preamble view does not. Assuming we want the header to get updated when a user clicks around on the table of contents within the preamble at some point, I don't think we can remove this line.